dist/ledger/README-cli.md

```diff
@@ -361,3 +361,146 @@ This concludes the upgrade process. The device is now ready to be used with the
 ## What's next
 
 Once the powHSM device is properly setup and onboarded, it is ready to be used with the powHSM middleware.
+
+## Troubleshooting
+
+This section lists some common issues that might arise during the setup and onboarding
+process and provides guidance on how to solve them.
+
+### Ledger Nano S screen is too dim
+
+Unfortunately, it is well a known problem that after a long time of usage, the
+Nano S screen might start dimming, eventually reaching a point where it is
+har to read the on-screen instructions. This is a hardware problem and there are
+some workarounds offered both by [Ledger](https://support.ledger.com/article/360021124674-zd)
+and [third-party websites](https://symetronix.com/ledger-nano-s-screen-not-working-comprehensive-guide-to-fix-the-issue/). 
+In any case, it is still possible to perform the process described in this document,
+even if the screen is completely unreadable. This section lists detailed steps so
+that the onboarding can be performed even without the capability of reading any
+of the on-screen instructions.
+
+For the update process, there's no need to boot the device on Recovery Mode.
+This means that the update can be performed just by reading the instructions
+on the terminal.
+
+#### Detailed steps for onboarding a new device
+
+This step is needed to onboard a new device. To access the Recovery Mode, follow
+the steps below:
+
+1. Press and hold the Right button on the Nano S. This is the button furthest
+   from the USB port. Keep it pressed while you connect the USB cable to the
+   device.
+
+2. Connect the USB cable to the device while holding the Right button pressed.
+   Keep it pressed for the next 5 seconds after the cable is connected, and then
+   release it. Wait for another 5 seconds before proceeding to the next step.
+
+> Now there are two ways to proceed, depending on whether the device was already
+  onboarded and has the Signer app installed. If the device is new, skip to step
+  4, otherwise, proceed to step 3.
+
+3. If the device was already onboarded and has the Signer app installed, the
```

Comment:
This is missing the step of entering the wrong pin three times to force a wipe before actually then going on to select the reset device option.